### DISCUSSION TITLE
Difficulties understanding solution behavior(TestAccept_BeforeAccept)

### DISCUSSION THREAD

**Discussion by SeminKim** [ANLAB-KAIST/KENSv3#85] [Project Q&A 2. Transport Layer (TCP)] [answered]
![image](https://user-images.githubusercontent.com/68637958/167436445-eccc6fd7-41d5-478a-a7cc-45ab6d43bfd0.png)

Hello, I found something strange while doing tests with some random seeds.
Above is the screenshot of `TestAccept_BeforeAccept` case in `test-kens-handshake-unreliable-run-solution`.
Here, I do not understand why Pkt No.7(ACK) is sent and what it is ACKing for, where 3-way handshake is not even finished yet.
Could you explain why this packet is sent?
Thanks.

**Comment by Pusnow** [comment, marked as answer]
It seems a bug in the solution binary.
You might dismiss the packet.

Sorry for the inconvenience.
